evennia/contrib/utils/debugpy/README.md

```diff
@@ -0,0 +1,157 @@
+# In-game debugpy command
+
+This registers an in-game command `debugpy` which starts the debugpy debugger and listens on port 5678.
+For now this is only available for Visual Studio Code (VS Code).
+
+If you are a JetBrains PyCharm user and would like to use this, make some noise at:
+https://youtrack.jetbrains.com/issue/PY-63403/Support-debugpy
+
+
+Credit for this goes to Moony on the Evennia Discord getting-help channel, thx Moony!
+
+0xDEADFED5 simply tied a pretty bow around it and stuck it here for everybody else.
+
```

Comment:
This intro must be on the correct form, since it's machine-read to build the Contrib overview pages. See [the contrib guidelines](https://www.evennia.com/docs/latest/Contribs/Contribs-Guidelines.html)